### ISSUE TITLE
App Store clone

### ISSUE THREAD

**Issue by diimdeep** [tuxu/nbviewer-app#9]
https://itunes.apple.com/ru/app/ipynb-viewer/id1400977081?l=en&mt=12

**Comment by tuxu** [comment]
Thanks for letting me know! This looks like a shameless ripoff...

**Comment by Jwink3101** [comment]
Certainly is shameless and a ripoff but this code is MIT licensed so it is allowed, right? (I am not a lawyer). Glad I found this before buying the app store one though

**Comment by tuxu** [comment]
We actually settled this already back in December. I haven't had a look at their actual app, but the developer assured me that it is actually an independent development. However, their app initially used the exact icons I created for Jupyter Notebook Viewer, which led to the confusion in the first place:
![appstore-ipynb](https://user-images.githubusercontent.com/147847/55264530-02045380-5275-11e9-9030-42b2ef642a28.png)

It's a very bad idea to ship an allegedly different app with the same app icon. Also, under the MIT license you are still required to preserve copyright and licensing notices.
